crates/key-value-azure/Cargo.toml

```
@@ -10,9 +10,9 @@ rust-version.workspace = true
 
 [dependencies]
 anyhow = { workspace = true }
-azure_data_cosmos = { git = "https://github.com/azure/azure-sdk-for-rust.git", rev = "8c4caa251c3903d5eae848b41bb1d02a4d65231c" }
-azure_identity = { git = "https://github.com/azure/azure-sdk-for-rust.git", rev = "8c4caa251c3903d5eae848b41bb1d02a4d65231c" }
-azure_core = { git = "https://github.com/azure/azure-sdk-for-rust.git", rev = "8c4caa251c3903d5eae848b41bb1d02a4d65231c" }
+azure_data_cosmos = { git = "https://github.com/azure/azure-sdk-for-rust.git", tag = "azure_data_cosmos-0.21.0"  }
```

Can we use the crates.io release instead?

---

Looks like 0.22.0 is available on crates.

---

I tried to update these changes to be on top of 0.22.0 when it came out a couple days ago but they removed some functionality for performing swaps using the `if-match` header. I filed an issue here: https://github.com/Azure/azure-sdk-for-rust/issues/2254

I can switch back to pointing to a commit sha if that is more robust

---

Is there some specific reason we need to use the repo here?

```suggestion
azure_data_cosmos = "0.21"
```

---

Sorry for the confusion here. I am not sure why i thought it wasn't published to crates. Updated